src/JavaMustKnowStuff/Task1.java

```diff
@@ -0,0 +1,64 @@
+package JavaMustKnowStuff;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task1 {
+
+    static void VowelOrNotVowel(String string){
+
+
+        Pattern p = Pattern.compile("[aeiouy]", Pattern.CASE_INSENSITIVE);
+
+        int length = string.length();
+        if (length % 2 == 0){
+
+            String[] split = string.split("");
```

Comment:
Let's rename this variable to 'letters'. It will be more clear what value it contains.